kfdefs/overlays/moc/smaug/opf-jupyterhub/alerts.yaml

```diff
@@ -26,3 +26,53 @@ spec:
             < 0.1
           labels:
             severity: critical
+    - name: SLOs-probe_success
+      rules:
+        - alert: RHODS Probe Success Burn Rate
```

Comment:
@mubariskhan96 RHODS  is an internal only service, so this alert name does not make sense.